FordLabs/PeopleMover
# 425 fix spacing error for long name

## Issue
Resolves a bug in #425 

## What was done
- [x] Fixed My Roles spacing

## How to test
1. Add a role that's really long, and see that it doesn't break the UI
2. Make sure no other features have been broken

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
